### DISCUSSION TITLE
Maps

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#2] [Ideas]
Maps are not completed yet. Record all the need zones across the whole map or record them from multiplayer sessions and for every animals is a long-term work.

Upon that, further updates will probably bring:
- points of interest
- map named zones
- custom markers
- maybe some quest related icons/positions
- contour lines

**Comment by giftigdegen** [comment]
Need to be able to mark the maps somehow.

**Reply by janstehno** [reply]
Yeah, the custom markers are one of the planned features for maps, so it will eventually be possible.

**Comment by Rancher17** [comment]
Could there be, say, 3-5 different types of custom makers? For example, I might want to Type 1 to mark where I shot a troll (so I know where to check again for potential respawns), Type 2 to mark a favorite hunting spot, Type 3 to mark the location of an animal I saw but did not shoot, etc.

**Comment by ammielynn** [comment]
I would even be willing to submit needs zones to the creators to get put on the app until we can add our own! Many of my needs zones have been in very different areas than what’s on the app!

**Reply by janstehno** [reply]
Yeah, why not. I am open to anything honestly. I have no time for the updates right now so this would help a lot in the meantime. If you need some map images for you to mark the locations, hit me up.

**Reply by angelmtztrc** [reply]
Hi, how can i submit missing need zones?

**Reply by janstehno** [reply]
Hello, the best possible way would be to screenshot the map with the zones and add an animal name description for each zone. I am not sure if there is a better way, as I do it similarly. I hover over each zone on the map and then make a recording of the last 3 minutes of what I did in the game. I then watch the recording and add the zones I hovered over to the app. I know it is time-consuming, but it is the best way by far to keep it as consistent and precise as possible.
